rfcs/20230323-add-split-graph-op/README.md

```diff
@@ -0,0 +1,161 @@
+# Add Split Operation in Graph API
+
+## Background
+### Motivation
+
+`Split` operation splits an input tensor into pieces of the specific length
+along some axis, which is opposite to the operation of `concat`. `Split`
+operation is supported by different deep learning frameworks and toolkit, eg.
+ PyTorch [1], TensorFlow [2], OpenVINO [3], and ONNX [4]. The `split`+`concat` 
+ pattern is more common in most models and takes up a large amount of computing 
+ time. However, since the graph API does not support `Split` op, possible 
+ optimization is limited. So we propose to add the `Split` op into graph API.
+
+### Investigation
+
+The `Split` operation is supported by most frameworks. To split an input tensor, 
+we need to specify the split dimension according to the `axis` for input tensor, 
+and then output multiple small chunks. But as for how to split along specific 
+dimensions, the definition of frameworks is slightly different, which usually 
+includes three rules. 
+- Specify the number of output tensors through an integer
+- Uniformly specify the size of each output chunk through an integer
+- Specify the specific size of each chunk through a list. 
+
+The size of all output chunks in the first two rules is the same but may differ 
```

Comment:
**Rule 2** above may lead to a tail chunk having a smaller shape than the specified shape (as was for the case of PyTorch and also specified in option 2), so please revise the statement here.

Reply:
ok, will do it.